instrumentation/p6spy/src/main/java/brave/p6spy/TracingP6SpyOptionsMBean.java

```diff
@@ -0,0 +1,11 @@
+package brave.p6spy;
+
+import com.p6spy.engine.logging.P6LogLoadableOptions;
+
+public interface TracingP6SpyOptionsMBean {
+    P6LogLoadableOptions getLogOptions();
```

Comment:
is this a stable api?

Reply:
p6spy requires all options classes to have an mbean interface, previously this was implicitly already added because P6SpyOptions was extended (I think incorrectly because none of its options were used) and it already exposed an mbean interface

Reply:
I mean to as if this P6LogLoadableOptions has been around for a while.. ex what version added etc. If we add this we could break others. Not all libraries track compatibility.

Reply:
hasn't been broken since the initial github release in 2013, 2.0.0